diktat-rules/src/main/kotlin/com/saveourtool/diktat/ruleset/utils/AstNodeUtils.kt

```diff
@@ -1186,28 +1196,33 @@ fun doesLambdaContainIt(lambdaNode: ASTNode): Boolean {
  * @return true if node has a pair
  */
 @Suppress("CyclomaticComplexMethod")
-internal fun isPairPropertyBackingField(propertyNode: ASTNode?, backingFieldNode: ASTNode?): Boolean {
-    val node = propertyNode ?: backingFieldNode
-    val propertyListNullable = (node?.treeParent?.getAllChildrenWithType(PROPERTY)) ?: return false
-    val propertyList: List<ASTNode> = propertyListNullable
-    val nodeType = node.getFirstChildWithType(KtNodeTypes.TYPE_REFERENCE)
-    val nodeNullableType = nodeType?.getFirstChildWithType(KtNodeTypes.NULLABLE_TYPE)
-    val nodeName = node.getFirstChildWithType(IDENTIFIER)?.text
+internal fun isPairPropertyBackingField(propertyNode: KtProperty?, backingFieldNode: KtProperty?): Boolean {
```

Comment:
May be add a default value: (backingFieldNode: KtProperty? = null)

Reply:
Nevermind, it's not symmetric

Reply:
Then function calls for property and backing field won't be symmetrical